tests/heading.js

```diff
@@ -91,5 +92,49 @@ describe( 'Heading', () => {
 				expect( dropdown.buttonView.label ).to.equal( 'Heading 1' );
 			} );
 		} );
+
+		describe( 'localization', () => {
+			let command;
+
+			beforeEach( () => {
+				const editorElement = document.createElement( 'div' );
+
+				const spy = testUtils.sinon.stub( Locale.prototype, '_t' ).returns( 'foo' );
```

Comment:
Since items are localized in the heading plugin before stubbed `editor.t` would work, this is the only way I see to test it. TBH, I don't like it and I wish we had some easier way to assert localization because it will repeat in many places.